Automattic/jetpack
# Boost: improve the display of scores in the dashboard

Fixes #20680

#### Changes proposed in this Pull Request:
Show previous score on Boost dashboard to display the improvement. If the scores improved, prompt for a review.

#### Jetpack product discussion
pbIAo1-hO-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Go to Boost Dashboard
- If you didn't already have any scores, it will generate the initial scores.
- After you have the initial score, try enabling Boost modules.
- Once the modules are enabled and ready, it should initiate auto-refresh within few seconds.
- If the score improved from the previous score for both Mobile and Desktop, you should see pointers for previous score as well as a card prompting to rate the plugin.
